### PR TITLE
docs: extract example configuration to a separate file

### PR DESCRIPTION
**Pull request by aboguszewski-sumo**

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Examples
```
Comment (perk-sumo) [inline docs/examples.md:1]: I'd rather call this file `configuration-examples.md`

File: docs/README.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ Documentation links:
     - [Container log parsing with Fluent-Bit (Docker, CRI-O, containerd)](/docs/fluent/container-logs.md)
     - [Performance estimates for running collection chart](/docs/fluent/performance.md)
     - [Fluentd persistence](/docs/fluent/fluentd-persistence.md)
+  - [Examples](/docs/examples.md)
```
Comment (perk-sumo) [inline docs/README.md:39]: This should actually be the first bullet under configuration and not last 😉

Comment (perk-sumo) [reply docs/README.md:39]: Also please rename the file to `configuration-examples.md`

File: docs/installation.md [hunk 3]
```diff
@@ -204,88 +201,7 @@ or exclude namespaces for Prometheus Operator installed with Sumo Logic Kubernet
...
-          privileged: true
-tailing-sidecar-operator:
-  scc:
-    create: true
-```
-
-**NOTE:** Please refer to
-[Using existing Operator to create Sumo Logic Prometheus instance](/docs/prometheus.md#using-existing-operator-to-create-sumo-logic-prometheus-instance)
-before applying the configuration.
+Examples can be seen in [this document](/docs/examples.md).
```
Comment (perk-sumo) [inline docs/installation.md:204]:
```suggestion
Examples can be seen in [this document](/docs/configuration-examples.md).
```

File: docs/installation.md [hunk 4]
```diff
@@ -204,88 +201,7 @@ or exclude namespaces for Prometheus Operator installed with Sumo Logic Kubernet
...
-
-An example file with the minimum configuration is provided below.
-
-```yaml
-sumologic:
-  accessId: ${SUMO_ACCESS_ID}
-  accessKey: ${SUMO_ACCESS_KEY}
-  clusterName: ${MY_CLUSTER_NAME}
-```
-
```
Comment (perk-sumo) [inline docs/installation.md:217]: I think I would actually keep this section and then after it wrote something like "for more configuration examples see this file"

File: docs/README.md (hunk 2 above)
Comment (perk-sumo) [inline docs/README.md:39]:
```suggestion
  - [Examples](/docs/configuration-examples.md)
```

File: docs/installation.md [hunk 5]
```diff
@@ -215,77 +213,7 @@ sumologic:
...
-          privileged: true
-tailing-sidecar-operator:
-  scc:
-    create: true
-```
-
-**NOTE:** Please refer to
-[Using existing Operator to create Sumo Logic Prometheus instance](/docs/prometheus.md#using-existing-operator-to-create-sumo-logic-prometheus-instance)
-before applying the configuration.
+More configuration examples can be seen in [this document](/docs/configuration-examples.md).
```
Comment (sumo-drosiek) [inline docs/installation.md:216]: Could we link the Openshift ones? As we don't have separate instruction for OpenShift, I think we should specifically say that Openshift examples are in examples

Comment (aboguszewski-sumo) [reply docs/installation.md:216]: You mean this one?
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/28b78fb8b5c0d19fb28cfd9376b280635c56bfc5/docs/installation.md#installing-in-openshift-platform

**Review by perk-sumo — APPROVED** [review]
🚀

File: docs/installation.md (hunk 5 above)
Comment (sumo-drosiek) [reply docs/installation.md:216]: Example was composition of few sections from installation section

Comment (sumo-drosiek) [reply docs/installation.md:216]: basically configuration you linked and prometheus one's, but it is not obvious for me that you need to go to examples to find complete openshift configuration

Comment (sumo-drosiek) [reply docs/installation.md:216]: or linking examples to openshift in `openshift installation` section

Comment (aboguszewski-sumo) [reply docs/installation.md]: I did the first one.

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**